src/manifest.ts

```diff
@@ -1005,13 +1006,18 @@ async function latestReleaseVersion(
       continue;
     }
 
-    return version;
+    if (version) {
+      logger.debug(
+        `Found latest release pull request: ${mergedPullRequest.number} version: ${version}`
+      );
+      candidateReleaseVersions.push(version);
```

Comment:
Will this behave appropriately with multiple branches, e.g., perhaps the tag is against a `preview` branch, but the release PR is against `main`?

Reply:
Yes, the first loop is iterating through commits on the target branch, so it should only find release PRs on that branch. Then we look at releases and only include that release if we see it's commit in the list of commits we've scanned (if we see one, then it's likely a later release than the one we found via PR). By cross-referencing against commit SHAs we've seen, we ensure we're only getting releases on the branch we are looking at.